readthedocs.yml

```diff
@@ -1,2 +1,7 @@
-conda:
-  file: doc/environment.yml
```

Comment:
We may need to reintroduce the `environments.yml` again at a later stage when redoing the documentation system with some advanced features, but for now I agree it is better to stick to the standard RTD build and standard dependencies.